Lib/fontParts/base/bPoint.py

```diff
@@ -1,16 +1,24 @@
-from fontTools.misc import transform
-from fontParts.base.base import (
-    BaseObject,
-    TransformationMixin,
-    SelectionMixin,
-    IdentifierMixin,
-    dynamicProperty,
-    reference
-)
-from fontParts.base.errors import FontPartsError
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, List, Optional, Tuple
+
 from fontParts.base import normalizers
+from fontParts.base.base import (BaseObject, IdentifierMixin, SelectionMixin,
```

Comment:
I see you're sticking to compact formatting rather than the original hanging indent style. Is this agreed upon somewhere? I have stuck to (and prefer) the original formatting, but I'll change to compact for consistency, if needed.

BTW, I employ this formatting  in most if not all other cases where a line break is necessary.

Reply:
I forgot the `isort` extension on vscode turned on. I will deactivate it for this workspace.